gap/congruences/conglatt.gd

```diff
@@ -7,26 +7,32 @@
 ##
 #############################################################################
 ##
-## This file contains functions for a lattice of congruences.
+## This file contains functions for a poset of congruences.
 ##
 ## When the congruences of a semigroup are computed, they form a lattice with
 ## respect to containment.  The information about how the congruences lie in
-## this lattice is stored in an IsCongruenceLattice object (a positional object
-## based on a list) and can be retrieved from this object with the following
+## this lattice may be stored in an IsCongruencePoset object (a component object
+## based on a record) and can be retrieved from this object with the following
 ## methods.
 ##
```

Comment:
Can you describe how the component object is organised? What are the components, how are they set?